scriptmodules/supplementary/kodi-extra.sh

```diff
@@ -4,10 +4,10 @@
 # For more information, please visit:
 #
 # https://github.com/RetroPie/RetroPie-Setup
-# https://github.com/Exarkuniv/RetroPie-Extra
+# https://github.com/ImAQwertyGuy/RetroPie-Extra
```

Comment:
If this would be merged into the main repo here, the original URL links should remain in-tact.

Reply:
if you can change it back
if you like you can add a line there with your name as the author so you get credit for it.

```suggestion
# https://github.com/Exarkuniv/RetroPie-Extra
```

Reply:
If you haven't noticed, this PR makes the script so it installs from the official Kodi repo rather than pipplware's outdated repo. It also gets rid of pipplware's repo from /etc/apt/sources.list.d. pipplware's repo works fine, except you can't use certain addons (like Invidious) on the version of Kodi it offers.

Reply:
he wasnt talking about what or where the program is installing from. he was talking about the highlighted line above his comment. that is all
the rest of it looks good

Reply:
also what version is this going to install? i tried it and it still installed 18.7 leia. so its not installing the latest if that's what it ment too.

also what Pi is this designed for. Pi5 or Pi4. i tested it on a pi 4 for now

Reply:
The first commit I made just deleted and added back a mark I think and I submitted by accident, so I did a new commit (the second one) that actually does what it's supposed to do.